Libraries/Opc.Ua.Client/Subscription.cs

```diff
@@ -1674,6 +1674,10 @@ private void OnKeepAlive(object state)
                     Utils.LogError(e, "Error while raising PublishStateChanged event.");
                 }
             }
+
+            // try to send a publish to recover stopped publishing.
+            int keepAliveInterval = (int)(Math.Min(m_currentPublishingInterval * m_currentKeepAliveCount, Int32.MaxValue));
+            m_session?.BeginPublish(Math.Min(keepAliveInterval, Int32.MaxValue / 3) * 3);
```

Comment:
Is the timeout parameter used in BeginPublish at all?

Reply:
I checked the code.. it is not used at this time, needs a follow up issue